lib/plugins/aws/info/getStackInfo.js

```diff
@@ -28,10 +29,24 @@ module.exports = {
           if (result) stackData.outputs = result.Stacks[0].Outputs;
         }),
     ];
-    if (this.serverless.service.provider.httpApi && this.serverless.service.provider.httpApi.id) {
+    const httpApiId =
+      this.serverless.service.provider.httpApi && this.serverless.service.provider.httpApi.id;
+    if (httpApiId) {
       sdkRequests.push(
-        this.provider
-          .request('ApiGatewayV2', 'getApi', { ApiId: this.serverless.service.provider.httpApi.id })
+        (httpApiId['Fn::ImportValue']
+          ? resolveCfImportValue(this.provider, httpApiId['Fn::ImportValue'])
+          : BbPromise.resolve(httpApiId)
+        )
+          .then(id => {
+            if (!id) {
+              throw new this.serverless.classes.Error(
+                `Could not resolve provider.httpApi.id parameter. Configured value: ${JSON.stringify(
+                  httpApiId
+                )}`
+              );
+            }
```

Comment:
Logically this should be attached to `resolveCfImportValue(this.provider, httpApiId['Fn::ImportValue']).then(..`